### ISSUE TITLE
Add hotkey binding for 'Copy Link Location'?

### ISSUE THREAD

**Issue by grnwood** [zim-desktop-wiki/zim-desktop-wiki#2615]
**tl;dr:**
Create a key binding for 'copy link to this location' when right clicking on a header allowing a keyboard only solution to copy the anchor link for pasting in another page.

**Use case:**
I use the Journal plugin DAILY to walk through each 'thing' I'm doing ... new meeting topic... new task I'm doing for project x/y/z.  I jump around so much in the day (like probably many other users of zim).

If I start something new, I type in a topic, hit ctrl-3 to give me a heading, then proceed to type in my notes/subpage links, etc.

This gives me a traceable record of 'what i did that day' Which looks like this:

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/1699293/934db908-13aa-42d0-9ae8-9e65749b510c)

However, at the end of the day or week I like to link those 'headings' into a more linear project tally like so:

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/1699293/e26bd73b-e165-41c6-9517-5d924a85ebba)

Which gives me a GREAT project log of what I've done.  

Headings, in general, and their associated anchor links are so useful in zim they deserve a first class key binding.  This way I could hover the text pointer onto any heading anywhere and hot key the 'copy link location' into the buffer for pasting into another page.  This would be a real win for zim in my opinion for my use case but many others (deep linking into headings).  The ToC Plugin also supports this link (which I used often) by right clicking.

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/1699293/01d6cde1-bfc6-4ae6-b949-1cca7a3e6c78)

I can try to do this myself if I get the time, but wanted to log it here either way.

**Comment by grnwood** [comment]
Duplicate of:

https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/2344
